tools/c7n_mailer/c7n_mailer/slack_delivery.py

```diff
@@ -200,8 +200,15 @@ def send_slack_msg(self, key, message_payload):
                 response.status_code, response.text)
             return
 
-        response_json = response.json()
-        if not response_json['ok']:
-            self.logger.info("Error in sending Slack message. Status:%s, response:%s",
-                             response.status_code, response_json['error'])
-            return
+        if response.headers['content-type'] == 'text/html':
+            if response.text != 'ok':
+                self.logger.info("Error in sending Slack message. Status:%s, response:%s",
+                                response.status_code, response.text)
```

Comment:
I'd recommend to log actualy response in 'debug' mode rather than 'info'. It can be pretty large (try to use wrong webhook)

Reply:
Here I'm just following logging approach(log level) seen through the project. Since the message is error it will not happen too often.

Reply:
yeah.. i think errors should be at least at info level or higher, so i'm fine this with this.